packages/styles/scss/components/feature-card/_feature-card.scss

```
@@ -80,7 +80,7 @@ $fcb-breakpoint-up--lg: map-get(
         }
       }
 
-      .#{$prefix}--card__content {
+      .bx--card__wrapper {
```

Minor nitpicking, but let's use the `$prefix` variable.

```suggestion
      .#{$prefix}--card__wrapper {
```